packages/office-ui-fabric-react/src/components/Breadcrumb/examples/Breadcrumb.Basic.Example.tsx

```diff
@@ -99,6 +99,16 @@ export const BreadcrumbBasicExample: React.FunctionComponent = () => {
         ariaLabel="Breadcrumb with maxDisplayedItems set to 2 and overflowIndex set to 1"
         overflowAriaLabel="More links"
       />
+      <Label styles={labelStyles}>With current item rendered as a heading</Label>
+      <Breadcrumb
+        items={[
+          { text: 'Files', key: 'Files', onClick: _onBreadcrumbItemClicked },
+          { text: 'This is folder 1', key: 'd1', onClick: _onBreadcrumbItemClicked },
+          { text: 'This is folder 2', key: 'd2', isCurrentItem: true, currentItemHeading: 'h2' }
```

Comment:
Feels strange to have this special case only for when an item is current.

Can we modify `IBreadcrumbItem` to follow a more common pattern instead? `as` is a commonly supported prop that would make sense here:

```
<Breadcrumb
        items={[
          { text: 'Files', key: 'Files', onClick: _onBreadcrumbItemClicked },
          { text: 'This is folder 1', key: 'd1', onClick: _onBreadcrumbItemClicked },
          { text: 'This is folder 2', key: 'd2', isCurrentItem: true, as: 'h2' }
```

This has the added benefit of supporting headings for non-current items if someone wanted to do that:

```
<Breadcrumb
        items={[
          { text: 'Files', key: 'Files', onClick: _onBreadcrumbItemClicked, as: 'h1' },
          { text: 'This is folder 1', key: 'd1', onClick: _onBreadcrumbItemClicked, as: 'h2' },
          { text: 'This is folder 2', key: 'd2', isCurrentItem: true, as: 'h3' }
```

Reply:
Definitely, I'll switch it over!